tuna/load_job.py

```diff
@@ -80,11 +81,11 @@ def parse_args():
                              help='add jobs with only these solvers '\
                                '(can be a comma separated list)')
   parser.add_argument(
-      '-o',
-      '--only_applicable',
-      dest='only_app',
+      '-d',
+      '--only_dynamic',
+      dest='only_dynamic',
```

Comment:
Make the 'load applicable solver' as default is a great idea which will avoid loading unnecessary solvers configs during the tuning process. The load_job script looks good to me.  @cderb, we need these changes in the staging branch for tuning on tuna_net2.